File: abc161/a/a.ts
import * as fs from "fs";

function swap(x: number, y: number, z: number) {
  console.log(z, x, y);
}

function main() {
  const input = fs.readFileSync("/dev/stdin", "utf-8").trim().split(/\s+/).map(Number);
  const x = input[0];
  const y = input[1];
  const z = input[2];
  swap(x, y, z);
}

main();

File: abc161/a/a.c
#include <stdio.h>

int swap(int x, int y, int z);

int main(void) {
  int x, y, z;
  scanf("%d %d %d", &x, &y, &z);
  swap(x, y, z);
  return 0;
}

int swap(int x, int y, int z) {
  printf("%d %d %d\n", z, x, y);
  return 0;
}
